app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,94 @@
+class TweetController < ApplicationController
+
+	get '/tweets' do 
+		if logged_in?
+			@tweets = Tweet.all
+			erb :'/tweets/tweets'
+		else
+			redirect "/login"
+		end
+	end
+
+	get '/tweets/new' do 
+		if logged_in?
+
+			if !session[:content] 
+				session[:content] = true
+				@content = false
+			end
+
+			erb :'/tweets/create_tweet'
+		else
+			redirect to "/login"
+		end
+	end
+
+	post '/tweets' do 
+		content = Tweet.new(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)